{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/apps/site/templatetags/site.py

```diff
@@ -196,7 +198,67 @@ def md(value, inline=True):
 def get_header_content():
     return Header.objects.first()
 
-
 @library.global_function
 def get_footer_content():
     return Footer.objects.first()
+
+@library.global_function
+@library.render_with('edit_bar.html')
+@jinja2.contextfunction
+def edit_bar(context):
+    context = dict(context)
+    request = context['request']
+
+    # Don't show to non-admins, and don't pretend that /search/ is editable.
+    if not request.user.is_staff or request.path == '/search/':
```

Comment:
Should this take in to account page edit permissions as well?

Reply:
It does on line 243.

Reply:
Whoops, misread what this line does 👍